docs-src/index.md

```diff
@@ -16,16 +16,13 @@ limitations under the License.
 
 # Introduction
 
-This is the documentation for the evolution of service-related APIs
-for Kubernetes. This project is part of the Kubernetes project working
-under [SIG-NETWORK][sig-network].
+Service APIs is the evolution of Kubernetes APIs that relate to services, e.g. Ingress,
```

Comment:
Nit: Capitalization feels inconsistent here. I think it should be `Services` if you're referring to Service resources and not a more generic concept.

Feel free to disregard this suggestion, but it might be worth splitting this up into two sentences:

```suggestion
This project represents the evolution of Kubernetes APIs that relate to Services, including the Ingress API. These APIs are an official Kubernetes [SIG-NETWORK][sig-network] subproject. 
```

Reply:
@robscott commit `cc70ce7` implements your feedback above.